std/typecons.d

```diff
@@ -796,35 +796,61 @@ if (distinctFieldNames!(Specs))
          *           for comparison between `Tuple`s.
          *
          * Returns:
-         * For any values `v1` on the right-hand side and `v2` on the
-         * left-hand side:
+         * For values `v1` on the left-hand side and `v2` on the right-hand side:
+         *
+         * 0 if the expression `v1 == v2` is true for all members, or else for
+         * the first member, where this is not true:
          *
          * $(UL
-         *   $(LI A negative integer if the expression `v1 < v2` is true.)
-         *   $(LI A positive integer if the expression `v1 > v2` is true.)
-         *   $(LI 0 if the expression `v1 == v2` is true.))
+         *   $(LI NaN, in case one of the operands is a NaN.)
+         *   $(LI A negative number if the expression `v1 < v2` is true.)
+         *   $(LI A positive number if the expression `v1 > v2` is true.))
```

Comment:
It would appear the `v1 == v2` case is no longer covered here.

Reply:
That's covered in line 801.

Reply:
In my defense, I'm an idiot. :p